applications/fishing-map/src/features/app/App.tsx

```diff
@@ -205,8 +206,8 @@ function App(): React.ReactElement {
       {/* <RecoilizeDebugger /> */}
       <Suspense fallback={null}>
         <SplitView
-          isOpen={sidebarOpen}
-          showToggle={showToggle}
+          isOpen={sidebarOpen && drawMode === 'disabled'}
```

Comment:
I don't like `drawMode` to be used in App when it's not a concern of App. Maybe another selector such as `selectIsDrawing` which would be a boolean.